static/docs/commands-reference/get.md

```diff
@@ -0,0 +1,45 @@
+# get
+
+Download or copy file or directory from another DVC repository into the local
```

Comment:
Should we clarify it a little bit:

`... DVC repository (usually, a link to Github or any other git server) ... `

or `... DVC repository (link to a Git server) ... `

or `... DVC repository in Git ...`

Reply:
Yes, good call. Updated in e00738e